extensions/ql-vscode/src/vscode-tests/no-workspace/query-results.test.ts

```diff
@@ -364,4 +490,12 @@ describe('query-results', () => {
     }
     return fqi;
   }
+
+  function safeDel(file: string) {
+    try {
+      fs.unlinkSync(file);
+    } catch (e) {
+      // ignore
+    }
+  }
```

Comment:
I have a PR up that removes this from `queries.test.ts`. As I explain above, can you use `del`?

Reply:
Hmmm...I had to remove `del` from my PR sine it was hanging. Apologies for the back and forth, but using `safeDel` here is fine.

Reply:
Ugh... still getting that Windows failure

```
[main 2022-10-20T20:56:10.030Z] Extension host with pid 756 exited with code: 134, signal: null.

Test runner caught exception (Failed)
Tried running suite 3 time(s), still failed, giving up.
Exit code:   134
Done
```
now with `safeDel`.

Reply:
That's annoying. Just for fun, can you try rebasing this PR on top of https://github.com/github/vscode-codeql/pull/1611?

Reply:
I just did — getting some errors locally (`Error: Cannot find module '/Users/angelapwen/repos/vscode-codeql/extensions/ql-vscode/out/vscode-tests/no-workspace/index'`) after rebasing when I try to debug now but pushed up to see if it's a local dev environment thing 🤷

Reply:
Same failure, except now I can't run it locally 😆 going to revert the rebase